libclc/utils/prepare-builtins.cpp

```diff
@@ -75,6 +75,17 @@ int main(int argc, char **argv) {
   if (NamedMDNode *OCLVersion = M->getNamedMetadata("opencl.ocl.version"))
       M->eraseNamedMetadata(OCLVersion);
 
+  // Drop wchar_size module flag
```

Comment:
Could you add clarification why we drop this flag, please? See a good example in the comments for the previous if-statement.

Reply:
Sure, would this be okay. FYI @steffenlarsen 

 // Drop wchar_size module flag. Windows adds the wchar_size variable 
  // which is not used in linux. libclc inherits the wchar_size flags 
  // from windows so this must be dropped before mangling. The wchar_size
  // module flag is taken out by the non-wchar_size flags are added back 
  // into the module.

Reply:
I think you could simplify it to:

> wchar_size flag can cause a mismatch between libclc libraries and modules using them. Since wchar is not used by libclc we drop the flag.

Reply:
Okay sounds good